custom/icds_reports/models/aggregate.py

```diff
@@ -347,6 +347,8 @@ class ChildHealthMonthly(models.Model, AggregateMixin):
     lunch_count = models.IntegerField(blank=True, null=True)
     state_id = models.TextField(blank=True, null=True)
     opened_on = models.DateField(blank=True, null=True)
+    birth_weight = models.PositiveSmallIntegerField(null=True, help_text="birth weight in grams")
+    mother_id = models.TextField(blank=True, null=True)
```

Comment:
I made this comment in the other PR but this is a very confusingly named column/indicator. The indicator actually refers to the person case id of the child, not the mother. If you intended this to be the person case of the mother, that is already in this table as `mother_case_id` and if you intend it to be that of the child, can you please rename it to be more clear.